ui/app/AppLayouts/Chat/views/ChatContextMenuView.qml

```diff
@@ -127,11 +128,11 @@ StatusPopupMenu {
         icon.name: "edit"
         enabled: root.isCommunityChat && root.isCommunityAdmin
         onTriggered: {
-            let popup = Global.openPopup(editChannelPopup, {
+            let popup = editChannelPopup.createObject(Global.applicationWindow, {
                                           isEdit: true,
                                           channelName: root.chatName,
                                           channelDescription: root.chatDescription
-        })
+            })
 
             popup.createCommunityChannel.connect(function(chName, chDescription){
```

Comment:
@alexandraB99 since you moved `editChannelPopup` here we can do a call here as it was:
```
Global.openPopup(editChannelPopup, {
    isEdit: true,
    channelName: root.chatName,
    channelDescription: root.chatDescription
})
```
and remove these connections:
```
popup.createCommunityChannel.connect
popup.editCommunityChannel.connect
popup.openPinnedMessagesPopup.connect
```

and add appropriate slots to `CreateChannelPopup`
```
Component {
        id: editChannelPopup
        CreateChannelPopup {
            anchors.centerIn: parent
            isEdit: true
            onClosed: {
                destroy()
            }

            onCreateCommunityChannel: root.createCommunityChannel(root.chatId, chName, chDescription)
            onEditCommunityChannel: root.editCommunityChannel(root.chatId, chName, chDescription)
            onOpenPinnedMessagesPopup: root.openPinnedMessagesList(root.chatId, chName, chDescription)
        }
    }
```

Reply:
Done